src/app/components/NewHeader/NewHeader.style.ts

```diff
@@ -13,6 +13,15 @@ export const useStyles = makeStyles(
       display: 'flex',
     },
 
+    logo: {
+      display: 'flex',
+
+      fontSize: theme.spacing(2.5),
+      [theme.breakpoints.up('tabletXS')]: {
+        fontSize: theme.spacing(5),
```

Comment:
Was this code copied? We've come to using pixels instead of theme.spacing for new components